### ISSUE TITLE
The Legend of Heroes: Ao no Kiseki KAI (1461920)

### ISSUE THREAD

**Issue by xilwen** [ValveSoftware/Proton#5513]
# Compatibility Report
- Name of the game with compatibility issues: The Legend of Heroes: Ao no Kiseki KAI
- Steam AppID of the game: 1461920

## System Information
- GPU: AMD Radeon RX 460
- Driver/LLVM version: Mesa 21.3.4
- Kernel version: 5.13.0-27-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/xilwen/d935835899ce66fccc155a9dbb41a120
- Proton version: experimental-6.3-20220119

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1461920.log](https://github.com/ValveSoftware/Proton/files/7920193/steam-1461920.log)

## Symptoms 

- The game won't launch due to unimplemented function `RegisterSuspendResumeNotification()` in `USER32.dll`.  
  Patch wine with `RegisterSuspendResumeNotification()` stubs makes the game launch.  
  PoC patch: [wine-RegisterSuspendResumeNotification-stub.diff.txt](https://github.com/ValveSoftware/Proton/files/7920224/wine-RegisterSuspendResumeNotification-stub.diff.txt)

- After the game launched with the above patch, it will crash right after a black window appeared;  
  can be workaround by turns "skip all movies" on in game options.  
  (Partially works in newer wine but still crashes the game due to some unimplemented functions)

- Controllers seems not being detected by the game with Steam Input enabled or disabled.

> Another game comes from the same series/publisher, `The Legend of Heroes: Zero no Kiseki KAI (1457520)` will also run into these issues.

## Reproduction

1. Set compatibility tool to `Experimental`, `6.3-8`, `5.13-6`, or even custom build `Proton-7.0rc6-GE-1` with newer wine
2. Launch the game, the game will quit immediatly without showing anything.

**Comment by xilwen** [comment]
Some updates:

- The first symptom has been fixed with `7.0-3-rc1` thanks to ValveSoftware/wine@0767dbe9;   
  this game can launch without crashing immediately now.
- Movies in game now plays but occasionally freeze right before the video finished with `7.0-3-rc1`.  
  The freezes seem to be gone away by checking `Use Inner VP9 Decoder` box in game settings tool.   
  When the video freezes, following logs can be observed:
```
!Err! New2 Register Miss

!Err! New2 Register Miss

... (repeats another 27 times)
```
   and sometimes:
```
winegstreamer warning: videobox0: not negotiated
winegstreamer warning: videobox0: ../src-gstreamer/libs/gst/base/gstbasetransform.c(1431): gst_base_transform_reconfigure_unlocked (): /GstBin:bin0/GstVideoBox:videobox0:
```
   
- Controller works well with **Steam Overlay** enabled, it's my bad to disable the overlay.

**Comment by xilwen** [comment]
Finished main story with `Proton: 1672857942 experimental-7.0-20230104`, video playback is now working well without freezing the game! All issues listed above have gone and the game runs really great, closing this issue.
